src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -136,6 +137,10 @@ public void ConfigureServices(IServiceCollection services)
             else
                 mvcBuilder = services.AddMvc();
 
+            services.AddOptions();
+
+            services.Configure<List<RoatpSequences>>(_configuration.GetSection("RoatpSequences"));
```

Comment:
How does this get injected to the relevant places? Sorry, just not seen it like this before

I'm used to this kinda code
```
Configuration = ConfigurationService.GetConfig(_config["EnvironmentName"], _config["ConfigurationStorageConnectionString"], Version, ServiceName).Result;
config.For<IWebConfiguration>().Use(Configuration);
```

Reply:
It's injected using IOptions<T>
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1
I've not included it in the Azure table storage config because it's not environment specific

Reply:
Fair enough. Might be worth a comment in the code saying that is being read from appsettings.json